src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/LaunchDarklyFunctionalTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.waworkflowapi;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.waworkflowapi.utils.LaunchDarklyClient;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class LaunchDarklyFunctionalTest extends SpringBootFunctionalBaseTest {
+
+    @Autowired
+    private LaunchDarklyClient launchDarklyClient;
+
+    @Test
+    public void should_hit_launch_darkly_and_return_true() {
+        boolean launchDarklyFeature = launchDarklyClient.getKey("tester");
```

Comment:
do we have this tester key in darkly? I cannot find it.

Reply:
Another question is,  can this test work in our local env? Since we don't have darkly configured in our local env, do we?

Reply:
You should just need the sdk-key stored locally in order to use the launchDarkly locally. I tested it this way and it works correctly.

Reply:
OK, cool. where can I get that key from?